spec/validators/account_number_validator_spec.rb

```diff
@@ -0,0 +1,62 @@
+require "rails_helper"
+class Validatable
```

Comment:
If possible, avoid defining classes like these in specs; when writing this way, they're not cleaned-up per test; since different spec files have `class Validatable`, due to Ruby open classes, the different test files are sharing one class, which is rather odd in my opinion.

If you can, try this approach:

https://github.com/codeforamerica/vita-min/blob/faed0d61efde3f616723e13c3505b38315b9adcc/spec/lib/submission_builder/formatting_methods_spec.rb#L3-L4

Not a blocker.